quickstart.sh

```diff
@@ -373,8 +432,16 @@ EOF
     setupAccountInteractive
   fi
 
+  if  [[ ! -e ${NETSTATS_ENV_FILE} ]]; then
```

Comment:
```suggestion
  if  [[ ! -f ${NETSTATS_ENV_FILE} ]]; then
```

Reply:
I think this is more precise and also what we actually need, especially 6 lines below.

Reply:
I've changed all of the -e tests...